src/custom-pv-components/pdb-navigation.js

```diff
@@ -17,25 +17,58 @@ const height = 40,
   };
 
 class ProtvistaPdbNavigation extends ProtvistaNavigation {
-  
+
   connectedCallback() {
     this.style.display = 'block';
     this.style.width = '100%';
     this.width = this.offsetWidth;
 
     this._offset = parseFloat(this.getAttribute('offset')) || 0;
```

Comment:
We have a lot of parseInt or parseFloat with getAttribute. What do you think of this.getIntAttribute / this.getFloatAttribute?